README.md

```diff
@@ -1,484 +1,13 @@
 # Bijectors.jl
 
-[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://turinglang.github.io/Bijectors.jl/stable)
+[![Latest](https://img.shields.io/badge/docs-latest-blue.svg)](https://turinglang.github.io/Bijectors.jl)
```

Comment:
Maybe keep a link to the latest release (stable) and the latest (possibly unreleased) commit (dev)?

Reply:
Unfortunately `stable` docs doesn't exist :confused:

Reply:
AFAICT the problem is that TagBot.yml contains a bug - the action does not have sufficient permissions to trigger other Github actions, and hence docs won't be built when a new tag/release is made.

Reply:
We should be able to fix this and then we could point to stable docs as well.

Reply:
Still not finding any stable docs :confused:

Reply:
Was a new release created after the TagBot PR was merged? I think only then a stable docs build should be triggered.

Reply:
Aaah, gotcha. Aight, I'll add it to the README then.